src/main/java/it/eng/idsa/businesslogic/configuration/SslValidatingConfiguration.java

```diff
@@ -0,0 +1,80 @@
+package it.eng.idsa.businesslogic.configuration;
+
+import java.security.KeyManagementException;
+import java.security.NoSuchAlgorithmException;
+import java.security.cert.CertificateException;
+import java.security.cert.X509Certificate;
+
+import javax.net.ssl.HostnameVerifier;
+import javax.net.ssl.HttpsURLConnection;
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.SSLSession;
+import javax.net.ssl.TrustManager;
+import javax.net.ssl.X509TrustManager;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class SslValidatingConfiguration {
+
+	private static final Logger logger = LogManager.getLogger(SslValidatingConfiguration.class);
+
+	public SslValidatingConfiguration(
+			@Value("${application.disableSslVerification:false}") boolean disableSslVerification) {
+		if (disableSslVerification) {
+			logger.info("Disabling ssl validation");
+			disableSslVerification();
+		} else {
+			logger.info("Using default ssl validation");
+		}
+	}
+
+	private void disableSslVerification() {
+		try {
+			// Create a trust manager that does not validate certificate chains
+			TrustManager[] trustAllCerts = new TrustManager[] { new X509TrustManager() {
+
+				@Override
+				public X509Certificate[] getAcceptedIssuers() {
+					// TODO Auto-generated method stub
```

Comment:
Can you remove these TODO comments, please?

Reply:
Yes sure, I removed!